### PR TITLE
Error handling for projects missing Queue Date

### PR DESCRIPTION
**Pull request by alneberg**
Checking if Queue Date is missing and will report "Missing" with a red danger-label if so.

### CODE REVIEW

**Comment by alneberg** [comment]
Live on stage as of now. If it works, you wont see any diffence as there aren't any project currently lacking queue date.

**Review by ssjunnebo — APPROVED** [review]
Looks good! 👍

File: status/worksets.py [hunk 1]
```diff
@@ -326,7 +326,11 @@ def get(self):
                 else:
                     total_num_samples = limsg.get_sample_number(projectlimsid=project)
                     proj = Project(limsg, id=project)
-                    date_queued = proj.udf['Queued'].strftime("%Y-%m-%d")
+                    try:
```
Comment (aanil) [inline status/worksets.py:329]: Hmm...

**Review by aanil — APPROVED** [review]
Maybe add a picture of your local where you tested it?

File: status/worksets.py (hunk 1 above)
Comment (alneberg) [reply status/worksets.py:329]: Do you mean "HMPF!?"

**Comment by alneberg** [comment]
@aanil I introduced the hack on dev instead. Now the only project available is listed with a None queue date.

**Comment by aanil** [comment]
Looks good! 👍

**Pull request merged**